packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -57,7 +58,7 @@ def test_stream(cluster):
         lines = response.iter_lines()
         sse_id = next(lines)
         assert sse_id, 'First line must be id. Got {}'.format(sse_id)
-        data = next(lines).decode()
+        data = next(lines).decode('utf-8', 'ignore')
```

Comment:
this won't work with `stream=True` because `response.text.iter_lines` will block
i think `.decode` is the only option we can use here